draft-ietf-quic-transport.md

```diff
@@ -906,6 +906,11 @@ explained in more detail as they are referenced later in the document.
 | 0x10 - 0x17 | STREAM            | {{frame-stream}}            |
 {: #frame-types title="Frame Types"}
 
+All QUIC frames are idempotent.  That is, the state of the connection is
+unchanged regardless of whether a frame is processed one time or multiple times,
+and a frame can be received and processed without error at any point between the
```

Comment:
Not all frames are processed without errors, so this is a bit awkward

Reply:
Perhaps that it never produces an error on a subsequent processing that it didn't produce the first time?